packages/plan-form/src/Form/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -69,20 +69,12 @@ Array [
   },
   Object {
     "errors": Array [
-      "'title' is required",
```

Comment:
Why is this one being deleted?

Reply:
@moshthepitt for both `name` and `title` the fields now have a  value, though the value is malformed. This means the fields do not fail the required  check anymore. however the `forward slash check` is applied only on title since name is dependent on title's value.

Reply:
Cool.  Is it important to maintain a test for the "required" validation?

Reply:
yeah its still being checked here: https://github.com/OpenSRP/web/blob/13128a44768420788cc15dedcf9cdb26a1f134e6/packages/plan-form/src/Form/tests/__snapshots__/index.test.tsx.snap#L7